packages/autorest.python/scripts/eng/pylintrc

```diff
@@ -9,15 +9,15 @@ ignore=_generated,samples,examples,test,tests,doc,.tox,generated_samples
 [MESSAGES CONTROL]
 
 # Add enable for useless disables
-enable=useless-suppression
+# enable=useless-suppression
```

Comment:
I fix the CI to enable pylint for autorest.python then some tiny lint issues are reporeted. I disable some specific items and we could fix them in another PR.